### DISCUSSION TITLE
Change skybox image on with some button click

### DISCUSSION THREAD

**Discussion by fmicozzi** [google/model-viewer#2704] [Q&A]
Anyone knows how to achieve this?

**Comment by elalish** [comment]
Follow this [example](https://modelviewer.dev/examples/scenegraph/#variants), but set `.skyboxImage` instead of `.variantName`.

**Reply by fmicozzi** [reply]
Thanks for your response but I don't know anything about programming :(

**Reply by elalish** [reply]
You'll need a little JS for this; I'd recommend doing a web tutorial to get the basics. :)
